### ISSUE TITLE
A11y_.Net Core_Winforms_MainForm_Dialog Controls_Color Dialog_Screen reader : Narrator Focus is not going to the individual Colors and it is not announcing the color names also. 

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [dotnet/winforms#9231]
### .NET version

Microsoft.Net SDK6.0.311 (x64)

### Did it work in .NET Framework?

No

### GitHubTags:
#A11yMAS;#A11yTCS;#BM_.NET Core[Winforms]-Win32-May2023;#.NET Core;#A11ySev2;#WCAG1.3.1;#DesktopApp;#Win11;#Benchmark;#Narrator;#NVDA;

### Actual:
The Narrator screen reader focus is not going to individual colors and it is not announcing the color name also.

**Note :** This Issue is also reproducing using NVDA screen reader

### Expected:
The Narrator/NVDA Screen reader focus should go to the Individual color blocks and it should announce the color name as well.

### User Impact:
Screen reader user will face difficulty in knowing and picking up the colors.

### Steps to reproduce

1. Install and Open Winforms App
2. Main Form screen would appear.
3. Tab till "Dialog Controls : Testing the Controls Under the Dialog Tab" and Hit enter.
4. Navigate to "Color Dialog" Button and Hit enter.
5. Observe with Screen reader that the Color Names are not announced by Narrator and the focus also does not move to the color blocks.

### Attachment:

**Narrator Attachment :** 
![Narrator Color](https://github.com/dotnet/winforms/assets/85863895/85b2c300-6e91-4ed1-818a-e90ee4d23742)

**NVDA Attachment:**
![NVDA color](https://github.com/dotnet/winforms/assets/85863895/ca783c1a-2e37-4d27-b0da-a712e6db7f4c)

**Comment by MelonWang1** [comment]
This issue same as [7352](https://github.com/dotnet/winforms/issues/7352). In comments,  @merriemcgaw  said this is CommonDialogs in Windows. And closed it when request exception approved [https://ceacc.visualstudio.com/ceaccex/_workitems/edit/130273](https://ceacc.visualstudio.com/ceaccex/_workitems/edit/130273).

**Comment by Ashutoshsahoo164** [comment]
As per the above mentioned comments closing this bug.
GitHubTags:#Closed;#A11yExceptionApproved;
